src/qpid/broker/SelectorExpression.cpp

```diff
@@ -1036,7 +1036,7 @@ Expression* unaryArithExpression(Tokeniser& tokeniser)
 Expression* parseExactNumeric(const Token& token, bool negate)
 {
     int base = 0;
-    string s;
+    string s("");
```

Comment:
Not necessary as std::string's default constructor already assigns an empty string.